src/main/java/nl/novi/techiteasyopnieuw/services/TelevisionService.java

```diff
@@ -0,0 +1,165 @@
+package nl.novi.techiteasyopnieuw.services;
+
+import nl.novi.techiteasyopnieuw.controllers.TelevisionController;
+import nl.novi.techiteasyopnieuw.dto.television.TelevisionDto;
+import nl.novi.techiteasyopnieuw.dto.television.TelevisionInputDto;
+import nl.novi.techiteasyopnieuw.exceptions.RecordNotFoundException;
+import nl.novi.techiteasyopnieuw.models.Television;
+import nl.novi.techiteasyopnieuw.repositories.RemoteControllerRepository;
+import nl.novi.techiteasyopnieuw.repositories.TelevisionRepository;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TelevisionService {
+    private final TelevisionRepository televisionRepository;
+    private final RemoteControllerRepository remoteControllerRepository;
+//constructor toevoegen om dependency injection uit te voeren
+
+
+    public TelevisionService(TelevisionRepository televisionRepository,
+                             RemoteControllerRepository remoteControllerRepository, RemoteControllerRepository remoteControllerRepository1) {
+        this.televisionRepository = televisionRepository;
+        this.remoteControllerRepository = remoteControllerRepository;
+    }
+
+    //functie ophalen van alle tv' + pas de Controller aan
+    //funtie ophalen 1  + pas de getmethode aan en maak een nieuwe methode in de service
+    //functie opslaan 1 tv
+    //functie verwijderen 1 tv
+    //functie updaten 1 tv
+
+    //alletv method
+
+    public List<TelevisionDto> getAllTelevisions() {
+        List<Television> tvList = televisionRepository.findAll();
+        List<TelevisionDto> tvListDto = new ArrayList<>();
+        //de nieuwe arraylist kan nu de nieuwe lijst opbergen in de DTO objects
+        for (Television t : tvList) {
+            tvListDto.add(televisionToTelevisionDto(t));
+        }
+        return tvListDto;
+    }
+
+    public TelevisionDto createTelevision(TelevisionDto createTelevisionDto) {
+        //maak een create aan of aparte methode moet overeenkomen met DTO variabelen
+
+        Television television = new Television();
```

Comment:
Je hebt hier duplicate code, Dit houdt in dat je ergens precies hetzelfde nog een keer doet, dat kan je ondervangen door er een aparte methode van te maken binnen je service :)